docs/pages/third_party_environments/index.md

```diff
@@ -7,6 +7,10 @@ title: Third Party Environments
 
 ## Video Game Environments
 
+### [ViZDoom](https://github.com/mwydmuch/ViZDoom)
+
+An environment around the original [Doom](https://en.wikipedia.org/wiki/Doom_(1993_video_game)) game, focusing on visual control (from image to actions) at thousands of frames per second. ViZDoom supports depth and automatic annotation/labels buffer, as well as accessing the sound. The Gym wrappers provide easy-to-use access to the example scenarios that come with ViZDoom. By 2022, the [ViZDoom paper](https://arxiv.org/abs/1605.02097) has been cited more than 600 times.
```

Comment:
> An environment around
Should this be "centered around" or smtg else?

Reply:
Yup, updated :)

Reply:
LGTM :)